tests/cisco/test_cisco_switch_protocol.py

```diff
@@ -194,7 +194,7 @@ def test_show_vlan_brief(self, t):
         t.readln("")
         t.readln("VLAN Name                             Status    Ports")
         t.readln("---- -------------------------------- --------- -------------------------------")
-        t.readln("1    default                          active    Fa0/2, Fa0/3, Fa0/4")
+        t.readln("1    default                          active    Fa0/2, Fa0/3, Fa0/4, Fa0/5, Fa0/6, Fa0/7, Fa0/8, Fa0/9, Fa0/10, Fa0/11, Fa0/12")
```

Comment:
Buuuuuut.... that's not how we assumed stuff up to now, this is made for testing automated tools, these tools usually use a defined screen size and may want to test with multiple lines in the port list